presto-native-execution/presto_cpp/main/common/Counters.h

```diff
@@ -114,6 +116,13 @@ constexpr folly::StringPiece kCounterOverloadedMem{"presto_cpp.overloaded_mem"};
 /// Worker exports 0 or 100 for this counter. 0 meaning not CPU overloaded
 /// and 100 meaning CPU overloaded.
 constexpr folly::StringPiece kCounterOverloadedCpu{"presto_cpp.overloaded_cpu"};
+/// Worker exports 0 or 100 for this counter. 0 meaning not overloaded and 100
```

Comment:
Is this just 0 or 1? Thanks!

Reply:
Changed it to 0/1 and COUNT type, as discussed.